### ISSUE TITLE
Build is failing:

### ISSUE THREAD

**Issue by AndKe** [jimevins/glabels-qt#101]
```
$ cmake ..
-- The C compiler identification is GNU 10.2.0
-- The CXX compiler identification is GNU 10.2.0
-- Check for working C compiler: /usr/lib/ccache/cc
-- Check for working C compiler: /usr/lib/ccache/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/lib/ccache/c++
-- Check for working CXX compiler: /usr/lib/ccache/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at cmake/Modules/FindGnuBarcode.cmake:38 (string):
  string sub-command REGEX, mode REPLACE needs at least 6 arguments total to
  command.
Call Stack (most recent call first):
  CMakeLists.txt:127 (find_package)


-- 
-- Project name ............ glabels-qt
-- Project version ......... 3.99-master561 (142aaf2 2020-03-28)
-- Installation prefix ..... /usr/local
-- Source code location .... /home/andre/Downloads/glabels-qt/glabels
-- CMake version ........... 3.16.3
-- C++ Compiler ............ GNU /usr/lib/ccache/c++ 10.2.0
-- Qt version .............. 5.12.6
-- zlib (optional).......... 1.2.11
-- GNU Barcode (optional)... 
-- qrencode (optional)...... 4.0.2
-- libzint (optional)....... No.
-- QtTest (optional)........ 5.12.6
-- 
-- Configuring incomplete, errors occurred!
See also "/home/andre/Downloads/glabels-qt/build/CMakeFiles/CMakeOutput.log".
```
LOG ATTACHED:
[CMakeOutput.log](https://github.com/jimevins/glabels-qt/files/5477833/CMakeOutput.log)

**Comment by randybb** [comment]
I have the same problem. It looks like there is a regexp that tries to get version of GNU Barcode from barcode.h, but at least in my barcode version there is no such string.

**Comment by sur5r** [comment]
Upstream dropped support for building the library. See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=914851

The Debian maintainer reintroduced support for it (see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=941997), but the header no longer contains the version number. So I guess the check in https://github.com/jimevins/glabels-qt/blob/master/cmake/Modules/FindGnuBarcode.cmake needs to be adjusted.

**Comment by KaiMartin** [comment]
Can I somehow disable bar code functionality altogether? I didn't need bar codes for the last five years. So probably won't miss it in the future.

With other applications that use cmake configure came up with a number of choices. However, in this case configure fails to finish gracefully.

**Comment by jimevins** [comment]
Should be fixed by 2da9b2d8565110dc02271d059a54b12b3d60b1ff.
